### PR TITLE
Added video processing section (Unit 7 - introduction )

### PR DESCRIPTION
**Pull request by jungnerd**
Added introduction of video processing and updated credits

### CODE REVIEW

File: chapters/en/unit7/video-processing/introduction-to-video.mdx [hunk 1]
```diff
@@ -36,3 +34,39 @@ Codecs, short for “compressor-decompressor” are software or hardware compone
...
+
+- **Storage Requirements:**
+High-resolution videos generate large volumes of data, leading to storage challenges. Efficient data compression and management techniques are necessary to handle the vast amounts of video data.
+
+- **Privacy and Ethical Concerns:**
+Video processing, especially in surveillance and healthcare, involves handling sensitive information. Ensuring privacy and addressing ethical concerns related to the misuse of video data are crucial considerations that must be carefully managed.
+
+## Conclusion
+
+Video processing is a dynamic and vital area within AI and CV, offering numerous applications and presenting unique challenges. Its importance in modern technology continues to grow, fueled by advancements in deep learning and the increasing availability of video data. In the following sections, we will dive deeper into the deep learning for video processing. You'll explore state-of-the-art models including 3D CNNs and transformers.
```
Comment (ATaylorAerospace) [inline chapters/en/unit7/video-processing/introduction-to-video.mdx:68]:
```suggestion
Video processing is a dynamic and vital area within AI and CV, offering numerous applications and presenting unique challenges. Its importance in modern technology continues to grow, fueled by advancements in deep learning and the increasing availability of video data. In the following sections, we will dive deeper into deep learning for video processing. You'll explore state-of-the-art models including 3D CNNs and transformers.  

```

File: chapters/en/unit7/video-processing/introduction-to-video.mdx [hunk 2]
```diff
@@ -36,3 +34,39 @@ Codecs, short for “compressor-decompressor” are software or hardware compone
 There are two main types of codecs; "lossless codecs" and "lossy codecs". Lossless codecs are designed to compress data without any loss of quality, while lossy codecs are more designed to compress by removing some of the data resulting in a loss of quality.
 
 In summary, a video is a dynamic multimedia format that combines a series of individual frames, audio, and often additional metadata. It is used in a wide range of applications and can be tailored for different purposes, whether for entertainment, education, communication, or analysis.
+
+## What is Video Processing?
+
+In the research field of computer vision (CV) and artificial intelligence (AI), video processing is all about automatically analyzing video data to understand and interpret both temporal and spatial features. Video data is simply a sequence of time-varying images, where the information is digitized both spatially and temporally. This allows us to perform detailed analysis and manipulation of the content within each frame of the video.
```
Comment (ATaylorAerospace) [inline chapters/en/unit7/video-processing/introduction-to-video.mdx:40]:
```suggestion
In the research field of computer vision (CV) and artificial intelligence (AI), video processing involves automatically analyzing video data to understand and interpret both temporal and spatial features. Video data is simply a sequence of time-varying images, where the information is digitized both spatially and temporally. This allows us to perform detailed analysis and manipulation of the content within each frame of the video.

```

**Review by ATaylorAerospace — CHANGES_REQUESTED** [review]
Thanks for adding this intro. Just a couple of small nits have been noted

**Comment by jungnerd** [comment]
> Thanks for adding this intro. Just a couple of small nits have been noted

Thanks for the review 😍

**Review by ATaylorAerospace — APPROVED** [review]

**Comment by jungnerd** [comment]
@merveenoyan Would you mind reviewing this for merge?

**Comment by cjfghk5697** [comment]
I'm not sure, but it seems like we should add this topic to the "toctree.yml".

File: chapters/en/unit7/video-processing/introduction-to-video.mdx [hunk 3]
```diff
@@ -36,3 +34,41 @@ Codecs, short for “compressor-decompressor” are software or hardware compone
 There are two main types of codecs; "lossless codecs" and "lossy codecs". Lossless codecs are designed to compress data without any loss of quality, while lossy codecs are more designed to compress by removing some of the data resulting in a loss of quality.
 
 In summary, a video is a dynamic multimedia format that combines a series of individual frames, audio, and often additional metadata. It is used in a wide range of applications and can be tailored for different purposes, whether for entertainment, education, communication, or analysis.
+
+## What is Video Processing?
+
+In the research field of computer vision (CV) and artificial intelligence (AI), video processing involves automatically analyzing video data to understand and interpret both temporal and spatial features. Video data is simply a sequence of time-varying images, where the information is digitized both spatially and temporally. This allows us to perform detailed analysis and manipulation of the content within each frame of the video.
```
Comment (johko) [inline chapters/en/unit7/video-processing/introduction-to-video.mdx:40]: To be consistent in the course, I think we should capitalize Computer Vision and Artificial Intelligence
```suggestion
In the research field of Computer Vision (CV) and Artificial Intelligence (AI), video processing involves automatically analyzing video data to understand and interpret both temporal and spatial features. Video data is simply a sequence of time-varying images, where the information is digitized both spatially and temporally. This allows us to perform detailed analysis and manipulation of the content within each frame of the video.

```

File: chapters/en/unit7/video-processing/introduction-to-video.mdx [hunk 4]
```diff
@@ -36,3 +34,41 @@ Codecs, short for “compressor-decompressor” are software or hardware compone
...
+
+## What is Video Processing?
+
+In the research field of computer vision (CV) and artificial intelligence (AI), video processing involves automatically analyzing video data to understand and interpret both temporal and spatial features. Video data is simply a sequence of time-varying images, where the information is digitized both spatially and temporally. This allows us to perform detailed analysis and manipulation of the content within each frame of the video.
+
+
+Video processing has become increasingly important in today's technology-driven world, thanks to the rapid advancements in deep learning (DL) and AI. Traditionally, DL research has focused on images, speech, and text, but video data offers a unique and valuable opportunity for research due to its extensive size and complexity. With millions of videos uploaded daily on platforms like YouTube, video data has become a rich resource, driving AI research and enabling groundbreaking applications.
```
Comment (johko) [inline chapters/en/unit7/video-processing/introduction-to-video.mdx:43]:
```suggestion
Video processing has become increasingly important in today's technology-driven world, thanks to the rapid advancements in Deep Learning (DL) and AI. Traditionally, DL research has focused on images, speech, and text, but video data offers a unique and valuable opportunity for research due to its extensive size and complexity. With millions of videos uploaded daily on platforms like YouTube, video data has become a rich resource, driving AI research and enabling groundbreaking applications.

```

File: chapters/en/unit7/video-processing/introduction-to-video.mdx [hunk 5]
```diff
@@ -36,3 +34,41 @@ Codecs, short for “compressor-decompressor” are software or hardware compone
...
+
+- **Storage Requirements:**
+High-resolution videos generate large volumes of data, leading to storage challenges. Efficient data compression and management techniques are necessary to handle the vast amounts of video data.
+
+- **Privacy and Ethical Concerns:**
+Video processing, especially in surveillance and healthcare, involves handling sensitive information. Ensuring privacy and addressing ethical concerns related to the misuse of video data are crucial considerations that must be carefully managed.
+
+## Conclusion
+
+Video processing is a dynamic and vital area within AI and CV, offering numerous applications and presenting unique challenges. Its importance in modern technology continues to grow, fueled by advancements in deep learning and the increasing availability of video data. In the following sections, we will dive deeper into deep learning for video processing. You'll explore state-of-the-art models including 3D CNNs and transformers.
```
Comment (johko) [inline chapters/en/unit7/video-processing/introduction-to-video.mdx:69]:
```suggestion
Video processing is a dynamic and vital area within AI and CV, offering numerous applications and presenting unique challenges. Its importance in modern technology continues to grow, fueled by advancements in deep learning and the increasing availability of video data. In the following sections, we will dive deeper into deep learning for video processing. You'll explore state-of-the-art models including 3D CNNs and Transformers.  

```

**Review by johko — CHANGES_REQUESTED** [review]
Thanks for contributing! I added some small proposals, actually all  capitalization stuff, for consistency within the course.

But great work :+1:

**Comment by jungnerd** [comment]
> Thanks for contributing! I added some small proposals, actually all capitalization stuff, for consistency within the course.
> 
> But great work 👍

Thanks for the review! I reflected your review and committed it!

**Review by johko — APPROVED** [review]
Thanks for the quick editing :)
Will merge now

**Comment by johko** [comment]
Looks like the quality check fails @jungnerd. Can you run `make style` from the root of your forked repository and commit the changed code? That should fix it

**Comment by jungnerd** [comment]
> Looks like the quality check fails @jungnerd. Can you run `make style` from the root of your forked repository and commit the changed code? That should fix it

@johko 
It looks like the Makefile does not contain a target for `style`, so `make style` fails. Should I add a style target to the Makefile to address this issue?

**Comment by johko** [comment]
> > Looks like the quality check fails @jungnerd. Can you run `make style` from the root of your forked repository and commit the changed code? That should fix it
> 
> @johko It looks like the Makefile does not contain a target for `style`, so `make style` fails. Can I add a style target to the Makefile to address this issue?

Ah yes, you're right. Strange, I thought it would be in there. I'll try to figure out what exactly make style should do and will either add it to he makefile myself and create a PR or tell you which command to run to fix the formatting @jungnerd

**Comment by johko** [comment]
I just created a PR which adds the command to the makefile: #324 
It also fixes the file which was not correctly formatted (turns out it slipped in through an other PR and is not your fault). 

As soon as my PR is merged, you can merge the changes into your branch and it should be good.

**Comment by jungnerd** [comment]
> I just created a PR which adds the command to the makefile: #324 It also fixes the file which was not correctly formatted (turns out it slipped in through an other PR and is not your fault).
> 
> As soon as my PR is merged, you can merge the changes into your branch and it should be good.

Great, thanks for checking 👍
I'll be waiting for your PR to be merged.

**Comment by johko** [comment]
@jungnerd I just merged the other PR (finally), so if you pull the upstream/main branch, the quality check should also work ;)

**Comment by johko** [comment]
nevermind, I could update it from here and will merge now :slightly_smiling_face:

**Pull request merged**